### DISCUSSION TITLE
Expand Profile Capacity

### DISCUSSION THREAD

**Discussion by LCzepiel** [runelite/runelite#18132] [Ideas]
I would love to see an expansion of the Profiles section on RuneLite. As a player with multiple profiles, having more space would be extremely convenient. Currently, the capacity is capped at 20 profiles. If it could be increased to around 40-50 profiles, that would be ideal.

**Comment by cdfisher** [comment]
Have you considered using something like "Plugin Presets" from the plugin hub rather than that many profiles?
